public_html/wp-content/plugins/wcpt/wcpt-loader.php

```diff
@@ -17,78 +18,78 @@
 define( 'WCPT_DIR', plugin_dir_path( __FILE__ ) );
 define( 'WCPT_URL', plugins_url( '/', __FILE__ ) );
 
-if ( !class_exists( 'WCPT_Loader' ) ) :
-/**
- * WCPT_Loader
- *
- * @package
- * @subpackage Loader
- * @since WordCamp Post Type (0.1)
- *
- */
-class WCPT_Loader {
-
+if ( ! class_exists( 'WCPT_Loader' ) ) :
```

Comment:
- [ ] Test if applications and admin view for Meetup and WordCamp loads properly.